### PR TITLE
⬆️ Use the Quarkus Artemis BOM and upgrade to 1.0.2

### LINKED ISSUE #3381: Use the Quarkiverse Artemis BOM and upgrade to 1.0.2
artemis versions has been moved to the io.quarkiverse.artemis:quarkus-artemis-bom, since 1.0.2

### PR DESCRIPTION
**Pull request by zbendhiba**
Fixes #3381

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -49,7 +49,7 @@
         <debezium.version>1.6.1.Final</debezium.version><!-- May go back to Camel's ${debezium-version} when they are in sync https://repo1.maven.org/maven2/io/debezium/debezium-bom/ -->
         <optaplanner.version>8.9.1.Final</optaplanner.version><!-- May go back to Camel's ${optaplanner-version} when they are in sync https://repo1.maven.org/maven2/org/optaplanner/optaplanner-quarkus/ -->
         <quarkiverse-amazonservices.version>1.0.0</quarkiverse-amazonservices.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/amazonservices/quarkus-amazon-services-parent/ -->
-        <quarkiverse-artemis.version>1.0.1</quarkiverse-artemis.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/artemis/quarkus-artemis-parent/ -->
+        <quarkiverse-artemis.version>1.0.2</quarkiverse-artemis.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/artemis/quarkus-artemis-parent/ -->
```
Comment (aldettinger) [inline pom.xml:52]: The comment here refers to quarkus-artemis-parent. Are we using it anymore ?

File: poms/bom-test/pom.xml [hunk 2]
```diff
@@ -54,6 +54,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>io.quarkiverse.artemis</groupId>
+                <artifactId>quarkus-artemis-bom</artifactId>
```
Comment (aldettinger) [inline poms/bom-test/pom.xml:59]: I was not able to find this in a maven repo. Could you please provide a link to have a look ?

Comment (zbendhiba) [reply poms/bom-test/pom.xml:59]: Oh wait, why did worked for me ?

Comment (aldettinger) [reply poms/bom-test/pom.xml:59]: I did not try to build @zbendhiba . I was just looking for the pom on sites like https://mvnrepository.com/search?q=quarkus-artemis-bom.

Comment (ppalaga) [reply poms/bom-test/pom.xml:59]: https://mvnrepository.com is a bit delayed. It is there in the Central https://repo1.maven.org/maven2/io/quarkiverse/artemis/quarkus-artemis-bom/1.0.2/quarkus-artemis-bom-1.0.2.pom

**Comment by oscerd** [comment]
Mvnrepository is not synched with the same timing of maven central. The source of Truth is always maven central

**Review by ppalaga — APPROVED** [review]
https://repository.jboss.org used to be flaky in the past. The CI failure is most probably unrelated.

> transfer failed for https://repository.jboss.org/nexus/service/local/repositories/thirdparty-releases/content/io/nessus/nessus-ipfs-client/1.0.0.Beta4/nessus-ipfs-client-1.0.0.Beta4.pom`

File: pom.xml (hunk 1 above)
Comment (zbendhiba) [reply pom.xml:52]: Maybe rename it to quarkus artemis. I'll push later the fix. Waiting for the nexus and artifacts to be available.
The Artemis parent, includes now the quarkus bom and quarkus-artemis-bom.  We don't refer to it in the code.

**Comment by aldettinger** [comment]
> https://repository.jboss.org used to be flaky in the past. The CI failure is most probably unrelated.
> 
> > transfer failed for [https://repository.jboss.org/nexus/service/local/repositories/thirdparty-releases/content/io/nessus/nessus-ipfs-client/1.0.0.Beta4/nessus-ipfs-client-1.0.0.Beta4.pom`](https://repository.jboss.org/nexus/service/local/repositories/thirdparty-releases/content/io/nessus/nessus-ipfs-client/1.0.0.Beta4/nessus-ipfs-client-1.0.0.Beta4.pom%60)

I agree, we are suffering the same kind of issue in another non related PR.

**Review by oscerd — APPROVED** [review]

**Pull request merged**